src/Features/VisualBasic/Portable/CodeRefactorings/VisualBasicRefactoringHelpersService.vb

```diff
@@ -57,5 +58,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings
         Public Shared Function IsIdentifierOfParameter(node As SyntaxNode) As Boolean
             Return (TypeOf node Is ModifiedIdentifierSyntax) AndAlso (TypeOf node.Parent Is ParameterSyntax) AndAlso (CType(node.Parent, ParameterSyntax).Identifier Is node)
         End Function
+
+        Protected Overrides Function TryGetVariableDeclaratorInSingleFieldDeclaration(node As SyntaxNode, ByRef singleVariableDeclarator As SyntaxNode) As Boolean
+            singleVariableDeclarator = Nothing
```

Comment:
I am surprised to find that currently no VB refactoring provider tries to find relevant VariableDeclarator node.
So just leave this part blank.

Reply:
I don't see a refactoring, but I do see this used in FAR: https://sourceroslyn.io/#Microsoft.CodeAnalysis.VisualBasic.Workspaces/VisualBasicSyntaxFacts.vb,662 

We should support this scenario for VB as well

Reply:
I added the VB side implementation. But I feel the link you posted might not be related to this bug? 
It's trying to fetch the VB VariableDeclarator inside the FieldDeclaration. (And here we are trying to fetch the VariableDeclarator from the FieldDeclaration if there is a single VariableDeclarator?)

Reply:
It might be related, and means we might need a fix there :)